src/amo/components/CollectionManager/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import "~photon-colors/colors";
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars.scss";
+
+.CollectionManager-slug {
+  // Always render the grid left-to-right. This UI involves a URL so
+  // it should always be displayed left-to-right.
+  direction: ltr;
+  display: grid;
+  grid-template-columns: auto auto;
+
+  input {
+    // Disregarding text direction is intentional.
+    border-left: none;
+    border-radius: 0 $border-radius-xs $border-radius-xs 0;
+    width: auto;
+  }
+}
+
+.CollectionManager-slug-url-hint {
+  background-color: $grey-20;
+  border: 1px solid $grey-50;
+  border-radius: $border-radius-xs 0 0 $border-radius-xs;
+  // Disregarding text direction is intentional.
```

Comment:
Adding "because this element is always LTR"  to these comments would clarify the why of this decision.

Reply:
It's not though. This one is RTL 😃  It's complicated.